app/controllers/application_controller.rb

```diff
@@ -8,6 +10,15 @@ class ApplicationController < ActionController::Base
 
   private
 
+  # Don't allow use of X-Forwarded-Host to prevent
+  # Host Header attack
+  # http://www.acunetix.com/blog/articles/automated-detection-of-host-header-attacks/
+  def drop_headers
+    if request.env['HTTP_X_FORWARDED_HOST']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.